mindmeld/active_learning/plot_manager.py

```diff
@@ -21,14 +21,14 @@
 from collections import Counter
 from typing import Dict, List
 import numpy as np
-import matplotlib.pyplot as plt
-
+from mindmeld.components._util import _is_module_available, _get_module_or_attr
```

Comment:
Can you check if this relative import works instead (for consistency)?

```suggestion
from ..components._util import _is_module_available, _get_module_or_attr
```